firmware/src/peripherals.cpp

```diff
@@ -148,9 +148,7 @@ void startPeripherals() {
   // Configure motor PWM timer and pause it
   pwmStart(&PWMD1, &motor_pwm_config);
   PWMD1.tim->CR1 &= ~TIM_CR1_CEN;
-  PWMD1.tim->CR1 = (PWMD1.tim->CR1 & ~TIM_CR1_CMS) 
-                   | TIM_CR1_CMS_0 | TIM_CR1_CMS_1 // (TIM1_CMS = 11) Enable center-aligned PWM
```

Comment:
Is there a reason center-aligned PWM can't be used? I was under the impression that it was better for harmonics, etc

Reply:
I agree center aligned is better for emissions and harmonics, but the trouble is that the reason it's better is that the transitions of the gates are not aligned.  We need to align the off transition of the gates so we can guarantee they're all off when we run our ISNS sampling.

It may be possible to do the ADC sampling in a more clever way, eg. injecting the isns channels after each gate driver transitions to low, which would allow use of the center aligned mode, but this would be a deeper re-architecting of the ADC driver that would take me a couple days, which is out of scope of the JITX contract for now.

Reply:
I sent an email with an explanation of the behavior change to David and Greg, would it be helpful to send to you as well?

Reply:
That'd be great! I'm not super involved with the firmware but am very curious about your work :)
`brentyi@berkeley.edu`